### PR TITLE
[improvement] improvement flink cluster alert

### PR DESCRIPTION
**Pull request by xujiangfeng001**
## What changes were proposed in this pull request

Issue Number: https://github.com/apache/incubator-streampark/issues/2865

## Brief change log

1. add `total number of jobs` information to the alarm

2. optimize the calculation of `affecting the number of jobs` in alarms

3. add `flink cluster state` display on the front end

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (no)

### CODE REVIEW

**Comment by xujiangfeng001** [comment]
cc @wolfboys @RocMarshal @wangsizhu0504  , PTAL,thx.

File: streampark-console/streampark-console-service/src/main/resources/mapper/core/ApplicationMapper.xml [hunk 1]
```diff
@@ -141,6 +141,16 @@
             limit 1
     </select>
 
+    <select id="countAffectedJobsByClusterId" resultType="java.lang.Integer" parameterType="java.lang.Long">
+        select
+            count(1)
+        from t_flink_app
+        where flink_cluster_id = #{clusterId}
+            and state in (5, 7, 9)
+            and (end_time IS NULL or TIMESTAMPDIFF(SECOND, end_time, NOW()) &lt;= 1)
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/resources/mapper/core/ApplicationMapper.xml]: Does TIMESTAMPDIFF support in h2 and pgsql ?

Comment (xujiangfeng001) [reply streampark-console/streampark-console-service/src/main/resources/mapper/core/ApplicationMapper.xml]: Thank you very much for the review. I have made changes to the code. Could you please review it when you have time.

Comment (xujiangfeng001) [reply streampark-console/streampark-console-service/src/main/resources/mapper/core/ApplicationMapper.xml]: > Does TIMESTAMPDIFF support in h2 and pgsql ?

Hi @wolfboys ，Thank you very much for the review. I have made changes to the code. Could you please review it when you have time.

File: streampark-console/streampark-console-webapp/src/enums/flinkEnum.ts [hunk 2]
```diff
@@ -140,11 +140,17 @@ export enum ClusterStateEnum {
...
   /** cluster started */
-  STARTED = 1,
+  RUNNING = 1,
   /** cluster canceled */
   CANCELED = 2,
   /** cluster lost */
   LOST = 3,
+  /** cluster unknown */
+  UNKNOWN = 4,
+  STARTING = 5,
```
Comment (wolfboys) [inline streampark-console/streampark-console-webapp/src/enums/flinkEnum.ts:150]: These values are too arbitrary.  We can set the status according to certain rules, such as transitioning from "create" to "start" to "stop." and then list some status of each stage, which would be better

File: streampark-common/src/main/java/org/apache/streampark/common/enums/ClusterState.java [hunk 3]
```diff
@@ -32,7 +32,7 @@ public enum ClusterState implements Serializable {
   /** cluster unknown */
   UNKNOWN(4),
   STARTING(5),
-  CANCELING(6),
+  CANCELLING(6),
```
Comment (wolfboys) [inline streampark-common/src/main/java/org/apache/streampark/common/enums/ClusterState.java:35]: These values are too arbitrary. We can set the status according to certain rules, such as from "create" to "start" to "stop", and then list some status of each stage,  e.g:
```  
CREATED(0),
STARTING(1),
RUNNING(2),
CANCELLING(3),
CANCELED(4),
FAILED(5),
KILLED(6);
LOST(7),
UNKNOWN(8)
```

Comment (xujiangfeng001) [reply streampark-common/src/main/java/org/apache/streampark/common/enums/ClusterState.java:35]: Due to historical status reasons, directly modifying the enumeration value may cause confusion in the historical status of the `flink cluster`. So I kept the original enumeration values.

Comment (wolfboys) [reply streampark-common/src/main/java/org/apache/streampark/common/enums/ClusterState.java:35]: > Due to historical status reasons, directly modifying the enumeration value may cause confusion in the historical status of the `flink cluster`. So I kept the original enumeration values.

thanks for the clarification

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**